k8s/namespaces/family-public-law/fpl-case-service/demo.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   values:
     java:
```

Comment:
Ideally you want to change
`fluxcd.io/tag.java: glob:pr-3059-*` to `fluxcd.io/tag.java: glob:pr-3193-*` as well

Reply:
updated.